base/shell/cmd/cmd.c

```diff
@@ -2308,6 +2309,20 @@ Initialize(VOID)
         ConOutPuts(_T("(C) Copyright 1998-") _T(COPYRIGHT_YEAR) _T(" ReactOS Team.\n"));
     }
 
+    /* ReactOS extension: Inform if the Asian font is not found */
+    hwndConsole = GetConsoleWindow();
+    if (GetPropW(hwndConsole, L"ReactOSFontFallback"))
+    {
+        ConOutPuts(_T("WARNING: The Asian fixed-pitch font not found. Falling back to English.\n"));
+
+        /* Set the thread locale to English */
+#define ENGLISH_LANGID MAKELANGID(LANG_ENGLISH, SUBLANG_ENGLISH_US)
+        SetThreadLocale(MAKELCID(ENGLISH_LANGID, SORT_DEFAULT));
+#undef ENGLISH_LANGID
+
+        RemovePropW(hwndConsole, L"ReactOSFontFallback");
+    }
```

Comment:
The problem with that is, what if a user doesn't run cmd.exe, but just run any other command-line utility?

Reply:
Trying some executables, I found it doesn't affect the other command-line utilities:
![VirtualBox_ReactOS_14_01_2022_08_38_38](https://user-images.githubusercontent.com/2107452/149425892-0e780359-5423-45a0-bb83-2f81cc9f938c.png)